src/Symfony/Bundle/FrameworkBundle/Tests/DependencyInjection/Fixtures/yml/messenger.yml

```diff
@@ -1,5 +1,6 @@
 framework:
     messenger:
+        stop_signals: SIGTERM
```

Comment:
Why does this have to be plural (a list of signals) instead of just 1 signal?

```suggestion
        stop_signal: SIGTERM
```

Reply:
in case project is used in several envs (docker FPM alpine environment, and classical VMs with supervisor) where signals could be different.

Reply:
Then configure the correct signal for that env. Doesn't make sense (to me) to just fire all the signals.

Reply:
Agreed with @Warxcell. we should allow multiple signals.